src/nvim/api/vim.c

```diff
@@ -1557,7 +1557,7 @@ void nvim_set_current_tabpage(Tabpage tabpage, Error *err)
 /// Creates a new namespace, or gets an existing one.
 ///
 /// Namespaces are used for buffer highlights and virtual text, see
```

Comment:
We should say extmarks instead. A virtual text is just a property of an extmark now.

Reply:
Dang it, I made that change in the doc txt file but not in the source file :/ Oops